OpenLiveWriter.Install.nuspec

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>openlivewriter</id>
+    <version>0.5.0.0</version>
+    <title>OpenLiveWriter (Install)</title>
+    <authors>See list of authors at http://openlivewriter.org/</authors>
+    <owners>Darwin Sanoy, OpenLiveWriter</owners>
+    <licenseUrl>https://github.com/OpenLiveWriter/OpenLiveWriter/blob/master/license.txt</licenseUrl>
+    <projectUrl>http://openlivewriter.org/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/csi-windowschocolatey/openlivewriter/master/app.png</iconUrl>
```

Comment:
You get bandwidth from rawgit for free.

Rawgit is not likely to be a dependency that would fail - but if it does so
will more than half the icons on Chocolatey.org

This is used to serve up your icon time someone sees your package in
chocolatey.org hit lists or when they view the individual listing.

Not sure when else (perhaps ChocolateyGUI and other graphic package
managers?).

D.



On Tue, Dec 15, 2015 at 2:56 PM, Martin Woodward <notifications@github.com>
wrote:

> In OpenLiveWriter.Install.nuspec
> <https://github.com/OpenLiveWriter/OpenLiveWriter/pull/75#discussion_r47688733>
> :
>
> > @@ -0,0 +1,26 @@
> > +﻿<?xml version="1.0"?>
> > +<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
> > +  <metadata>
> > +    <id>openlivewriter</id>
> > +    <version>0.5.0.0</version>
> > +    <title>OpenLiveWriter (Install)</title>
> > +    <authors>See list of authors at http://openlivewriter.org/</authors>
> > +    <owners>Darwin Sanoy, OpenLiveWriter</owners>
> > +    <licenseUrl>https://github.com/OpenLiveWriter/OpenLiveWriter/blob/master/license.txt</licenseUrl>
> > +    <projectUrl>http://openlivewriter.org/</projectUrl>
> > +    <iconUrl>https://cdn.rawgit.com/csi-windowschocolatey/openlivewriter/master/app.png</iconUrl>
>
> What are the requirements for the iconURL? Shall we stick a copy in the
> Open Live Writer CDN to save taking a dependency?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenLiveWriter/OpenLiveWriter/pull/75/files#r47688733>
> .
>